src/services/__mocks__/projectService.ts

```diff
@@ -0,0 +1,43 @@
+import { rejects } from 'assert';
+
+class ProjectService {
+  path = '/projects';
+
+  /**
+   * Gets all Projects.
+   *
+   * @returns Projects
+   */
+  public getProjects() {
+    return new Promise((resolve, rejects) => {
+      import('../../services/__mocks__/projects-mock.json').then((mockProjectsRequest) => {
+        resolve({ data: mockProjectsRequest });
```

Comment:
I guess there will be the same pattern (lines 12-16) often used also on the other places like `buildServices` etc., we can consider creating new util method (let's say `requestMock()`) for it so that it could look like:

```ts
public getProjects() {
   return requestMock('../../services/__mocks__/projects-mock.json');
}
```

Reply:
will be solved in NCL-6855